Applications/Spire/Source/Ui/CalendarDatePicker.cpp

```diff
@@ -14,6 +14,38 @@ using namespace boost::signals2;
 using namespace Spire;
 using namespace Spire::Styles;
 
+namespace {
+  auto make_header_label(QString text, QWidget* parent) {
+    auto label = make_label(std::move(text), parent);
+    label->setFixedSize(scale(24, 24));
+    auto style = get_style(*label);
```

Comment:
Declare at point of use. This jumps from ```style``` to ```font``` and then back to ```style```. Keep things local. Same above, you could do ```font``` then ```label``` then ```style```.